modules/va_facilities/spec/requests/nearby_request_spec.rb

```diff
@@ -209,7 +223,7 @@ def query_params(url)
     end
 
     it 'paginates according to parameters' do
-      setup_pdx
+      create_list(:generic_vha, 10)
```

Comment:
Fancy! I didn't know `create_list` was a thing!

Reply:
I saw `create_list` being used to generate an arbitrary number of VBA facilities and knew I had to steal the pattern.